compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5449,34 +5447,6 @@ istoreHelper(TR::Node * node, TR::CodeGenerator * cg, bool isReversed)
             translatedNode = translatedNode->getFirstChild();
          if (translatedNode->getOpCode().isRightShift()) // optional
             translatedNode = translatedNode->getFirstChild();
```

Comment:
This code involving `translatedNode` seems to have no use. Can we remove it? Similarly in the other codegens.